[ID1000323] implemente the details of organizations

Nixiangyu opened the pull request:
--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

hukunge commented on models/impl/org.go:
```diff
@@ -42,22 +43,22 @@ func (b *DbBackendImpl) QueryOrganizations(params *structs.QueryParams) ([]*stru
-	if err := where.Raw(sqlQueryDonationStatAndAccountInfo, "org", time.Unix(params.StartTime, 0),
-		time.Unix(params.EndTime, 0), params.PageLimit, offset).Scan(&out).Error; err != nil {
+	if err := where.Raw(sqlQueryDonationStatAndAccountInfo, time.Unix(params.StartTime, 0),
+		time.Unix(params.EndTime, 0), "org", params.PageLimit, offset).Scan(&out).Error; err != nil {
```
需要用const中的常量，

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

hukunge commented on models/impl/org.go:
```diff
@@ -68,5 +69,28 @@ func (b *DbBackendImpl) QueryOrganizations(params *structs.QueryParams) ([]*stru
+
+	var out structs.OrganizationDetailItem
+	where := b.GetConn().Model(&structs.OrganizationDetailItem{})
+	if err := where.Raw(sqlQueryDetailDonationStatAndAccountInfo, "org", uid).Scan(&out).Error; err != nil {
```
需要用const中的常量

hukunge commented on controllers/org/org.go:
```diff
@@ -55,3 +59,30 @@ func (h *RestHandler) QueryOrganizations(c *gin.Context) {
+
+	item, err := h.srvcContext.DBStorage.QueryOrganizationDetail(req.UID)
+	if err != nil {
+		e := fmt.Errorf("query organizations detail error : %s", err.Error())
```
逗号

hukunge commented on structs/org.go:
```diff
@@ -8,6 +8,7 @@ package structs
 
 import (
 	"github.com/shopspring/decimal"
+	"time"
```
time在上面

hukunge commented on structs/org.go:
```diff
@@ -40,3 +47,22 @@ type QueryOrganizationsResp struct {
 }
+
+// QueryOrganizationDetailRequest defines the request of query organization information
+type QueryOrganizationDetailRequest struct {
```
这个命名太长了，建议用Org缩写

hukunge commented on models/impl/org.go:
```diff
@@ -68,5 +69,28 @@ func (b *DbBackendImpl) QueryOrganizations(params *structs.QueryParams) ([]*stru
+			return nil, e
+		}
+
+		logger.Errorf("query organizations detail record error: %v", err)
```
逗号

Nixiangyu committed:
Merge pull request #2 from csiabb/master

[ID1000326] detail information of funds and supplies (#24)

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com

zhangbing16 commented on structs/org.go:
```diff
@@ -29,6 +31,12 @@ type OrganizationsItems struct {
+
+// ConvertTime defines the covert of created_at
+func (ois *OrganizationsItems) ConvertTime() {
+	ois.CreatedAt = ois.Time.Unix()
```
前后端交互的时间统一都用int64格式，时间值采用UTC时间吧

Nixiangyu replied:
好的

Commit:
[ID1000323] implemente the details of organizations

--story=1000323 --user=Nixiangyu
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000323

Signed-off-by: Nixiangyu Nixiangyu@ofworld.com
